README.md

```diff
@@ -21,19 +21,34 @@ The stryker-jest-runner is a plugin for Stryker to enable Jest as a test runner.
 For the minimum supported versions, see the peerDependencies section in package.json.
 
 ## Configuration
-Set your test runner to `jest` in the stryker config, optionally provide the project you're working on for a preset configuration.
-We currently only support "react" preset, enable it by adding the `jest` config property to your project and specify the `project` property.
-Also make sure you set your `testRunner` to "jest" and disable `coverageAnalysis`.
+Make sure you set the `testRunner` option to "jest" and set `coverageAnalysis` to "off" in your Stryker configuration.
 
 ```javascript
-jest: {
-    project: 'react',
-    testRunner: 'jest',
+{
+    testRunner: 'jest'
     coverageAnalysis: 'off'
 }
 ```
 
-### Loading the plugin
+Stryker-jest-runner provides a couple of configurable options using the `jest` property in your stryker config:
+
+```javascript
+{
+    jest: {
+        project: 'react',
+        config: require('path/to/your/custom/jestConfig.js')
+    }
+}
+```
+
+| option | value |
+|----|----|
+| project (optional) | The project you are working on (we currently only support "react")|
```

Comment:
This isn't true, as we also support "Default" right? Probably we should also explain what "react" does (mimic `react-scripts`)

Reply:
Well they're both optional, if you don't specify any of them it will load the config in your package.json. could clarify that.

Reply:
I know it is optional, but if it has only one option, i would assume it is also the default. So specifying there are 2 options will help.